modules/common/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/AtomicFields.scala

```diff
@@ -43,11 +43,11 @@ object AtomicFields {
       AtomicField(name = "app_id", _.app_id, _.app_id = null),
       AtomicField(name = "platform", _.platform, _.platform = null),
       AtomicField(name = "event", _.event, _.event = null),
-      AtomicField(name = "event_id", _.event_id, _.event_id = null),
+      AtomicField(name = "event_id", _.event_id, _ => ()),
       AtomicField(name = "name_tracker", _.name_tracker, _.name_tracker = null),
       AtomicField(name = "v_tracker", _.v_tracker, _.v_tracker = null),
-      AtomicField(name = "v_collector", _.v_collector, _.v_collector = null),
-      AtomicField(name = "v_etl", _.v_etl, _.v_etl = null),
+      AtomicField(name = "v_collector", _.v_collector, _ => ()),
+      AtomicField(name = "v_etl", _.v_etl, _ => ()),
```

Comment:
```suggestion
      AtomicField(name = "v_collector", _.v_collector, _ => ()), // required in loading
      AtomicField(name = "v_etl", _.v_etl, _ => ()), // required in loading
```

Reply:
add clarifying comment for v_collector and v_etl